tests/e2e/utils/Logger.ts

```diff
@@ -11,12 +11,48 @@ import { TestConstants } from '../TestConstants';
  **********************************************************************/
 
 export abstract class Logger {
+
+    /**
+     * Uses for logging of fatal errors.
+     * @param text log text
+     */
+    public static error(text: string) {
+        console.log(`    [FATAL] ${text} [FATAL]`);
```

Comment:
Could we use ```throw new Error('text')``` and get reed of this method?

Reply:
I wouldn't do that. That would cause an exception to be echoed along with a stacktrace. That's not necessarily the purpose of this logging message. This is for when an unrecoverable event happened and you want to log it in the place, so that you can see where it happened in the test output before you throw an actual exception

Reply:
e.g.:
```
  ...
  catch(error) {
    Logger.error(`SomeClass.someMethod Expected element disappeared unexpectedly. Cannot recover, ending test suite.`);
    throw new RuntimeException(`Expected element disappeared unexpectedly`, error);
  }
```

Reply:
Sorry, I can't get your mind.
Let me describe what I think. In a case, if something went really wrong you just have to interrupt execution by throwing an error message.
Here you can see how it works:
```
suite('Login test', async () => {
    test('Login', async () => {
        throw new Error('Custom error message');
        
        await driverHelper.navigateToUrl(TestConstants.TS_SELENIUM_BASE_URL);
        await loginPage.login();
    });
});

![Screenshot from 2020-03-10 14-38-43](https://user-images.githubusercontent.com/22724619/76313159-1e11d100-62dd-11ea-9411-f8a117d9c01a.png)


On the screenshot, you can see that an "Error" applies a custom message and also keep a stack trace.

Reply:
![Screenshot_20200310_135708-2](https://user-images.githubusercontent.com/16451875/76314678-91b0df80-62d7-11ea-9657-ba9591f8303d.png)

Reply:
with the above example I know when, within which method the fatal message occured.
At the bottom I only have the class name and suite name

Reply:
Maybe it's unnecessary, but I just liked the option to have a full spectrum of the methods.

Reply:
Sorry, in my eyes it looks like duplication. You can do all the same with pure "Error" customization.

Reply:
I tend to agree with Tibor here.
Logging something with "error" level and throwing the error/exception are two different things.

Also.. I would prefer the format to have `[ERROR]` instead of `[FATAL]`

Reply:
@Ohrimenko1988 well, with your solution, the message could be completely lost if somebody puts the method in a try-catch block one level higher. This way you are sure to have an error in the logs no matter what else happens in the code.

@rhopp Well, I intended the error level to only be used with fatal failures which cannot be recovered. I could change it if you really want, or we could look for a symbol that would represent an error instead

Reply:
@rhopp added a symbol for the exception instead of text in brackets